tests/unit/test_pre_processor.py

```diff
@@ -45,6 +46,19 @@ def test_resize():
     assert resized_mask.shape[2] == 3
 
 
+def test_divide_255():
+    orig_image = np.full((1, 1, 3), 1)
```

Comment:
To initialize array with 1, I prefer to use `np.ones`

Reply:
fixed. Thanks!